plugin.xml

```diff
@@ -12,22 +12,35 @@
+        <engine name="cordova-android" version=">=4"/>
+        <engine name="cordova-plugman" version=">=4.2.0"/><!-- needed for gradleReference support -->
     </engines>
 
+    <!-- android -->
     <platform name="android">
         <config-file target="res/xml/config.xml" parent="/*">
-            <preference name="webView" value="org.crosswalk.engine.XWalkWebViewEngine" />
+            <preference name="webView" value="org.crosswalk.engine.XWalkWebViewEngine"/>
+
+            <!-- Crosswalk Version -->
+            <!-- Example: '10', '15+', 'xwalk_core_library_beta:9+', 'org.xwalk:xwalk_core_library_beta:18+' -->
+            <!-- Reference: https://crosswalk-project.org/documentation/downloads.html -->
+            <preference name="CROSSWALK_ANDROID_VERSION" default="13+"/>
+
+            <!-- Crosswalk Arguments-->
+            <!-- Default: '––disable-pull-to-refresh-effect' -->
+            <!-- Reference: https://github.com/crosswalk-project/crosswalk-website/wiki/Crosswalk-Command-Line-Options -->
+            <preference name="CROSSWALK_ANDROID_COMMANDLINE" default="––disable-pull-to-refresh-effect"/>
         </config-file>
 
-        <source-file src="src/android/XWalkWebViewEngine.java" target-dir="src/org/crosswalk/engine" />
-        <source-file src="src/android/XWalkExposedJsApi.java" target-dir="src/org/crosswalk/engine" />
-        <source-file src="src/android/XWalkCordovaResourceClient.java" target-dir="src/org/crosswalk/engine" />
-        <source-file src="src/android/XWalkCordovaUiClient.java" target-dir="src/org/crosswalk/engine" />
-        <source-file src="src/android/XWalkCordovaView.java" target-dir="src/org/crosswalk/engine" />
-        <source-file src="src/android/XWalkCordovaCookieManager.java" target-dir="src/org/crosswalk/engine" />
+        <source-file src="platforms/android/src/org/crosswalk/engine/XWalkWebViewEngine.java" target-dir="src/org/crosswalk/engine"/>
+        <source-file src="platforms/android/src/org/crosswalk/engine/XWalkExposedJsApi.java" target-dir="src/org/crosswalk/engine"/>
+        <source-file src="platforms/android/src/org/crosswalk/engine/XWalkCordovaResourceClient.java" target-dir="src/org/crosswalk/engine"/>
+        <source-file src="platforms/android/src/org/crosswalk/engine/XWalkCordovaUiClient.java" target-dir="src/org/crosswalk/engine"/>
+        <source-file src="platforms/android/src/org/crosswalk/engine/XWalkCordovaView.java" target-dir="src/org/crosswalk/engine"/>
+        <source-file src="platforms/android/src/org/crosswalk/engine/XWalkCordovaCookieManager.java" target-dir="src/org/crosswalk/engine"/>
+
+        <framework src="platforms/android/xwalk.gradle" custom="true" type="gradleReference"/>
 
-        <framework src="src/android/xwalk.gradle" custom="true" type="gradleReference" />
+        <hook type="after_plugin_install" src="hooks/after_plugin_install/000-apply_cli_options.js"/>
```

Comment:
Make sense. it will be removed when the feature is available in cordova-lib stable.